content/blog/2019/08/2019-08-17-role-strategy-performance-folder-auth.adoc

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Performance Improvements to Role Strategy Plugin and Folder Auth Plugin"
+tags:
+- plugins
+- performance
+- gsoc
+- gsoc2019
+author: abhyudayasharma
+---
+
+The task for my link:/projects/gsoc/2019/role-strategy-performance[Google Summer of Code Project]
+was to improve the performance of the plugin:role-strategy[Role Strategy Plugin].
+I had started by creating a link:/blog/2019/06/21/performance-testing-jenkins[framework]
+for running micro-benchmarks for Jenkins plugins. I had started by creating Docker
+image for the simulation of the performance slowdown in the plugin followed by
```

Comment:
This sentence looks to be from the previous version of the post. It was the application period, and right now it looks a bit off. Just remove it or reword?

Reply:
Removed the section about performance improvements to Role Strategy.